{{cookiecutter.service_name}}/service.py

```diff
@@ -575,8 +575,8 @@ def {{cookiecutter.workflow_id |replace("-", "_")  }}(conf, inputs, outputs): #
         delete_configmap(v1, f"pod-env-vars-{job_id}", executing_namespace)
 
         # Deactivate workspace API tokens for both calling and executing workspace
-        deactivate_api_token(inputs.get("CALLING_WORKSPACE_TOKEN")["value"], "CALLING_WORKSPACE_TOKEN")
-        deactivate_api_token(inputs.get("EXECUTING_WORKSPACE_TOKEN")["value"], "EXECUTING_WORKSPACE_TOKEN")
+        deactivate_api_token(inputs.get("CALLING_WORKSPACE_ACCESS_TOKEN")["value"], "CALLING_WORKSPACE_ACCESS_TOKEN")
+        deactivate_api_token(inputs.get("WORKSPACE_ACCESS_TOKEN")["value"], "EXECUTING_WORKSPACE_ACCESS_TOKEN")
```

Comment:
```suggestion
        deactivate_api_token(inputs.get("CALLING_WORKSPACE_ACCESS_TOKEN", {}).get("value"), "CALLING_WORKSPACE_ACCESS_TOKEN")
        deactivate_api_token(inputs.get("WORKSPACE_ACCESS_TOKEN", {}).get("value"), "EXECUTING_WORKSPACE_ACCESS_TOKEN")
```

Haven't checked this but would something like this be a good idea in case WORKSPACE_ACCESS_TOKEN or CALLING_WORKSPACE_ACCESS_TOKEN aren't set? You're using `.get` for `inputs` anyway

Reply:
Good point, will update